tests/integration/test_code_distribution_query.py

```diff
@@ -1,6 +1,11 @@
+import os
+os.environ["TASK_API_BASE_URL"] = "https://localhost:8000"
```

Comment:
IS this meant to be here? Shouldn't it be set by the environment running the test?